src/modules/common/Config.ts

```diff
@@ -418,6 +434,67 @@ export class ConsensusConfig implements IConsensusConfig {
     }
 }
 
+/**
+ * Votera config
+ */
+export class VoteraConfig implements IVoteraConfig {
+    /**
+     * The address to which we bind
+     */
+    address: string;
+
+    /**
+     * The port on which we bind
+     */
+    port: number;
```

Comment:
Does it need both?

Reply:
it does not need the both but I think Interfaces should be design for flexible Extension of code in future.